client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -21,4 +24,24 @@ const PluginSite = ( props ) => {
 	return <PluginSiteJetpack { ...props } />;
 };
 
-export default PluginSite;
+export default connect(
+	( state, ownProps ) => {
```

Comment:
I think anon function is getting long enough that we can pull this out to `mapStateToProps `